jobs/integr8ly/pds-general.yaml

```diff
@@ -0,0 +1,300 @@
+---
+
+- job:
+    name: pds-general
+    project-type: pipeline
+    description: "General pipeline for PDS testing."
+    sandbox: false
+    concurrent: true
+    properties:
+      - build-discarder:
+          num-to-keep: 56
+    parameters:
+        - string:
+            name: YOURCITY
+            description: "City or Customer (5 char min.) plus the generated hash, e.g. qebrno-5d10 [required]"
+        - string:
+            name: GH_CLIENT_ID
+            description: "GitHub AOuth Client ID, required for some of the walkthroughs. Can be left empty"
```

Comment:
Small typo here :-) Sorry didn't notice that in my first review

Reply:
No worries will fix that